src/scss/css.scss

```diff
@@ -143,4 +144,12 @@
         cursor: pointer;
     }
 
+}
+
+@media screen and (min-width: $breakpoint-small) {
+    .apexcharts-reset-zoom-icon,
+    .apexcharts-zoom-icon {
+        width: 55px
```

Comment:
These two classes are used on the div containers around the svg images. They do not change the size of the image itself.

Reply:
The icons look bigger though.

Reply:
Just rechecked it and you are right. It does look bigger.